rxandroid/src/main/java/rx/android/widget/SearchViewQueryText.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package rx.android.widget;
+
+import android.widget.SearchView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.WeakHashMap;
+
+import rx.Observable;
+import rx.Subscriber;
+import rx.Subscription;
+import rx.android.AndroidSubscriptions;
+import rx.android.internal.Assertions;
+import rx.functions.Action0;
+
+
+public class SearchViewQueryText {
```

Comment:
There's no value in this class. Just pass `SearchView` to the methods and have them be static.